app/views/docs/database.phtml

```diff
@@ -0,0 +1,216 @@
+    To find specific documents in a collection, pass an array of query strings as a parameter to the <a href="/docs/server/database#databaseListDocuments">listDocuments</a> endpoint. The SDKs provide a <b>Query</b> class to make query building simpler:
+</p>
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Query } from "appwrite";
+
+sdk.database.listDocuments('movies', [
+    Query.equal('title', 'Avatar')
+]);
+</code></pre>
+</div>
+
+<p>The following operators are currently supported:</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Operator</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>equal</td>
+            <td>Equal to.</td>
+        </tr>
+        <tr>
+            <td>notEqual</td>
+            <td>Not equal to.</td>
+        </tr>
+        <tr>
+            <td>lesserEqual</td>
+            <td>Less than or equal to.</td>
+        </tr>
+        <tr>
+            <td>greater</td>
+            <td>Greater than.</td>
+        </tr>
+        <tr>
+            <td>greaterEqual</td>
+            <td>Greater than or equal to.</td>
+        </tr>
+        <tr>
+            <td>search</td>
+            <td>Requires a Fulltext Index.</td>
+        </tr>
+    </tbody>
+</table>
+
+<p>Each query string is logically separated via AND. For OR logic, pass multiple values, separated by commas:</p>
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>
+sdk.database.listDocuments('movies', [
+    Query.equal('title', 'Avatar', 'Lord of the Rings'),
+    Query.greater('year', 1999)
+]);
+</code></pre>
+</div>
+
+<h2><a href="/docs/advanced-database#indexes" id="indexes">Indexes</a></h2>
+<p>
+    Indexes are used by databases to quickly locate data without having to search through every document for results.
+    To ensure the best performance, Appwrite requires an index for every query.
+    You can create an index by navigating to the <b>Indexes</b> tab of your collection, or by using your favorite Server SDK.
+</p>
+<p>
+    The following indexes are currently supported:
+</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Key</td>
+            <td>Plain Index to allow queries.</td>
+        </tr>
+        <tr>
+            <td>Unique</td>
+            <td>Unique Index which not allows duplicates.</td>
+        </tr>
+        <tr>
+            <td>Fulltext</td>
+            <td>For searching within string attributes</td>
+        </tr>
+    </tbody>
+</table>
```

Comment:
```suggestion
</table>
<h3>Pagination</h3>
<p>Appwrite has full support for pagination to better optmise and scale up your applications built on appwrite. Detailed documentation on pagination and how to implement it can be found <a href="/docs/pagination#offset-pagination" id="offset-pagination">here.</a></p>
```